### PR TITLE
feat: add sashido & filter_bean logo under beta sponsors

### PR DESCRIPTION
**Pull request by padmajabhol**
### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your default branch!
- [x] Make sure you are making a pull request against the **default branch** (left side). Also you should start *your branch* off *default branch*.
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.
- [x] I have added necessary documentation (if appropriate)

### Description
Please describe your pull request.

❤️Thank you!

### Post merge checklist
- [ ] Follow steps from the [guidelines for contributing](../CONTRIBUTING.md) to this repository.
- [ ] If you are a new contributor, ping in the thread and one of the maintainers will add you to all-contributors list.

### CODE REVIEW

File: config/content/Sponsors.js [hunk 1]
```diff
@@ -137,6 +145,13 @@ export default {
       link: 'https://www.voiceflow.com/',
       alt: 'Voiceflow',
     },
+    {
+      src: 'https://res.cloudinary.com/dscnitrourkela/image/upload/project-oregano/sponsors/delta/jzu4m46qrkuheumwzrcx.png',
```
Comment (abhibhaw) [inline config/content/Sponsors.js:149]: This logo looks so ugly 🤣 try once without background!!

**Review by abhibhaw — CHANGES_REQUESTED** [review]

File: config/content/Sponsors.js (hunk 1 above)
Comment (padmajabhol) [reply config/content/Sponsors.js:149]: @ORKO06 do update the cloudinary link.

**Comment by riteshsp2000** [comment]
@ORKO06 @padmajabhol 
If there are some logos with backgrounds and not good looking, use [remove.bg](https://remove.bg)  to automatically remove the background, it works like a charm.

Also for delta sponsors, move the leading-learners logo at the very end. Its looking a bit abrupt (the gap between those)

**Comment by DesignrKnight** [comment]
The PR name says it's to add Shasido logo but you are adding two

Do update PR name

**Comment by padmajabhol** [comment]
> The PR name says it's to add Shasido logo but you are adding two
> 
> Do update PR name
 I made a separate commit for the other sponsor though if that's not okay, I'll keep the PR thing in mind next time.
https://github.com/dscnitrourkela/project-oregano/pull/92/commits/f658023882cb2dbb304cc1cd7cd8567ec07daed8

**Review by riteshsp2000 — APPROVED** [review]

**Review by riteshsp2000 — APPROVED** [review]

**Pull request merged**